class/world.lua

```diff
@@ -98,6 +111,12 @@ World = Class {
             enemy:draw()
         end
 
+        love.graphics.setColor(1, 1, 0) -- floating moneys 
+        for i, gain in pairs(self.floatingGains) do 
+            love.graphics.print("+" .. gain.yield, gain.worldOrigin.x, gain.worldOrigin.y)
+            gain.worldOrigin.y = gain.worldOrigin.y - 0.5
```

Comment:
ooo one last thing, the REAL reason we dont update in draw() is because we dont have access to delta time, which means any logical update will not be consistent cross machines/frame rates.